taxi/views.py

```diff
@@ -4,11 +4,9 @@
 from django.urls import reverse_lazy
 from django.views import generic
 from django.contrib.auth.mixins import LoginRequiredMixin
-
 from .models import Driver, Car, Manufacturer
-from .forms import DriverCreationForm, DriverLicenseUpdateForm, \
-    CarForm, DriverSearchForm, CarSearchForm, \
-    ManufacturerSearchForm
+from .forms import DriverCreationForm, DriverLicenseUpdateForm, CarForm
+from .forms import DriverSearchForm, CarSearchForm, ManufacturerSearchForm
```

Comment:
Why do you have dublicated import.  @Abnormaltype wanted that you instead of `/` use `()`
`from .form import (
    DriverCreationForm, DriverLicenseUpdateForm, ......
)`